public/dist/js/restaurant-week.app.js

```diff
@@ -293,6 +272,51 @@
 })(angular.module('rwApp'));
 'use strict';
 
+(function (app) {
+	var restaurantListDirective = function restaurantListDirective(constants) {
+		var directive = {
+			restrict: 'E',
+			scope: {
+				list: '=',
+				filtermodel: '='
```

Comment:
Not going to block, but this should really be `filterModel` since it's two words.